exercises/concept/pacman-rules/test/Tests.hs

```diff
@@ -0,0 +1,68 @@
+import PacmanRules (eatsGhost, loses, scores, wins)
+import Test.Hspec (describe, hspec, it, shouldBe)
+
+main :: IO ()
+main = hspec $ do
+  describe "eatsGhost" $ do
+    it "ghost eaten when touched while powered" $ do
```

Comment:
and inconsistency that I decided wasn't important enough to resolve, but could be: this test uses the phrase "while powered" and "while un-powered", whereas tests for `loses` and `wins` do not use the word "while"; they just use the words "powered" and "un-powered" alone.

Reply:
I like the `while`. I went ahead and updated and force-pushed.

Reply:
All right! While reading that message I realised there's something implicit in there that I overlooked: as you can see, I'm 100% comfortable with having branches force-pushed even while I potentially am working on them and have them checked out. And I was used to doing that when everything should be squashed, since then nobody has to remember to recheck the message when merging. But I realised not everyone might be the same way. For example I think it's sort of a breach of etiquette for me to have done it without having asked first. So I know it's late, but sorry for doing that without notice.